go.mod

```diff
@@ -352,6 +352,7 @@ require (
 	github.com/siddontang/go v0.0.0-20180604090527-bdc77568d726 // indirect
 	github.com/siddontang/go-log v0.0.0-20180807004314-8d05993dda07 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
+	github.com/stretchr/objx v0.5.0 // indirect
```

Comment:
🤔 I don't understand how this indirect import appeared

Reply:
using `testify/mock`